### PR TITLE
Base hardcoded content for the new disco pane (#2449)

### PR DESCRIPTION
**Pull request by diox**
This allow translators to start working on the strings while we build an API (the initial implementation will likely be stored that way, and the content not change very much at first).

### CODE REVIEW

**Comment by muffinresearch** [comment]
@diox are you thinking you'll do the substutions your end or will it be the client that does that?

**Comment by diox** [comment]
@muffinresearch what do you prefer ? I keep going back and forth on this in my mind. On the one hand it makes sense for the client to do it so that it can change the HTML at will, on the other `<blockquote>` and `<cite>` are already there on the server side...

**Comment by muffinresearch** [comment]
Yeah I feel similarly on the fence. I think it makes sense to do it in the same place (on the server) for now. We can always revisit it again later.

**Comment by diox** [comment]
Ok, wfm.

File: src/olympia/discovery/data.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+from olympia import amo
+
+
+class DiscoItem(object):
+    def __init__(self, *args, **kwargs):
+        self.addon_id = kwargs.get('addon_id')
+        self.type = kwargs.get('type')
+        self.heading = kwargs.get('heading')
+        self.description = kwargs.get('description')
```
Comment (EnTeQuAk) [inline src/olympia/discovery/data.py]: This could really just be a namedtuple right? Or are any of those items potentially optional?

**Comment by EnTeQuAk** [comment]
Given the fact that the disco pane is probably one of the most looked at aspects of AMO I feel like we should add an information (l10n comment) about those strings for translators so that they know precisely what they're doing and where the stuff happens that they're translating.

Especially if there's some marketing speech involved.

r+wc

File: src/olympia/discovery/data.py (hunk 1 above)
Comment (diox) [inline src/olympia/discovery/data.py]: I started with a `namedtuple`, but then realized I'm going to need different properties for themes, plus some might be completely optional even for extensions, so I used a plain object instead.

**Comment by diox** [comment]
What kind of comment did you have in mind ? I thought about it but I couldn't come up with something that was really worth having as a comment.

File: src/olympia/discovery/data.py (hunk 1 above)
Comment (EnTeQuAk) [inline src/olympia/discovery/data.py]: fair enough

**Comment by EnTeQuAk** [comment]
oh wait, the file-name already has the discovery part in it and you cannot group strings and add one comment like "The following strings are shown in the Firefox discovery pane, when you click on about:addons". Something like that is what I was thinking about but it'd be as a comment on every single string and that is probably overkill.

Maybe we can do that if we really have to. Not sure if we have any best practices for that kind of stuff, I can just imagine it must be hard sometimes to reason about where this string is being used, the context and everything.

**Pull request merged**